main.py

```
@@ -10,7 +10,8 @@
 CB_API_KEY         = os.environ.get("COINBASE_API_KEY", "")
 CB_WEBHOOK_SECRET  = os.environ.get("COINBASE_WEBHOOK_SECRET", "")
 BASE_URL           = os.environ.get("BASE_URL", "https://apex-revenue-system.up.railway.app")
-CUSTOMERS_FILE     = "/tmp/customers.json"
+REVENUE_LEDGER_FILE = "/tmp/revenue_ledger.json"
```

Ruff (0.16.2)

[error] Probable insecure usage of temporary file or directory: "/tmp/revenue_ledger.json"

(S108)